compiler/luci-interpreter/src/kernels/FullyConnected.test.cpp

```diff
@@ -26,26 +26,56 @@ namespace
 
 using namespace testing;
 
-TEST(FullyConnectedTest, Float)
+// Returns the corresponding Bias DataType given the type T.
+template <typename T> constexpr DataType getBiasType()
```

Comment:
There is `getElementType` in `TestUtils.h`, please use it.

Reply:
On this, only float input case return for float, and quantize case for S32. so, I think this is little bit different.

Reply:
I see. In this case the comment shold tell about that (that the given type T is the input type, not the bias type).

Reply:
Sorry for confusing. I will update more clearly.